setup.cfg

```diff
@@ -1,2 +1,6 @@
 [metadata]
-description-file = README.rst
\ No newline at end of file
+description-file = README.rst
+
+[options.extras_require]
+hdf5 =
+    tables>=3.7.0 # Pandas 2.0 has an hdf5 variant... use that later?
```

Comment:
do not worry, we restrict the pandas pacakge requirements to <2. We can ignore this part.

Reply:
@SphtKr